### PR TITLE
Mobile version finished.

### PR DESCRIPTION
**Pull request by EmiLoKiJu**
- Add the form with functionalities
- Add style and positioning for the section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,6 +180,61 @@ <h1 class="aboutmeh1">
...
+                  <span class="linespan"></span>
+                  <li>
+                    <textarea id="msg" name="user_message" placeholder="Write me something..." maxlength="500" required></textarea>
+                  </li>
+                  <li class="submitform">
+                    <button type="submit">Get in touch</button>
+                  </li>
+                  
+                </ul>
+            </form>
```
Comment (Eternalgratis) [inline index.html]: - Kindly remove the ul and li tags in the form. These are basically used as bullets to list items but in this case, you won't be needing them. The inputs field are enough

File: stetics.css [hunk 2]
```diff
@@ -410,12 +410,130 @@ body {
...
+  margin-right: 6.7%;
+  margin-bottom: 5%;
+}
+
+.formlu {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0%;
+  list-style-type: none;
+}
```
Comment (Eternalgratis) [inline stetics.css:443]: - Please increase the width of the input field so its looks like the one in the Figma.

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
Hi @EmiLoKiJu , 

Good job so far!
There are some issues that you still need to work on in order to go to the next project, but not to worry, you are almost there!

## Highlights
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally



## Required Changes ♻️
 Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise. **

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: stetics.css (hunk 2 above)
Comment (EmiLoKiJu) [reply stetics.css:443]: image

Can you be more specific? I don't see any difference.

Comment (EmiLoKiJu) [reply stetics.css:443]: @Eternalgratis

**Comment by gersonhiggins** [comment]
image
Would be nice to place images into a folder, cause work with a orderly repository would look better and make it easier to find thing on it.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,5 @@
 <a name="readme-top"></a>
```
Comment (gersonhiggins) [inline README.md:2]: kindly add a descriptive introduccion about your project, so the people can see what this project about

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @EmiLoKiJu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
A Pull request should be descriptive. It should have:
1. a descriptive title telling you what changes are inside.
2. a short summary describing introduced changes in more detail.

image

These good practices become our habits to do clean work that results in good collaboration with other fellow developers. 

_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EmiLoKiJu** [comment]
@RaoAkif I just edited what you did request. Do you think is ok now?

Have a nice day~

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Form works as intended 👍 
- Form validations added 👍 
- Linter set up well and checks pass 🟢
- PR description is descriptive 📝

## Optional suggestions
⚠️  Please use descriptive branch names. FBranch doesn't tell about anything about the feature you added on the branch. "setup-contact-from", and "contact-form" would be better naming. [Check](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) more to learn more about this.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**